drivers/sensor/tdk/icp101xx/icp101xx_drv.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023 TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_ICP101XXX_ICP101XXX_H_
```

Comment:
There doesn't seem to be a good reason to split this into a .h file, unless there's multiple .c files needing this it should be in the same compliation unit to keep things simple